.github/workflows/release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: npm test
   update-api-definitions-changelog:
     runs-on: ubuntu-latest
-#     needs: [test]
+    needs: [test]
```

Comment:
Would we also need to enable the pre-commit [hook](https://github.com/twilio/twilio-cli/blob/main/githooks/pre-commit)?

Reply:
Thanks for pointing! Yeah makes sense, we can add back the pre-commit hook, which'd also close https://issues.corp.twilio.com/browse/DII-175